Translations/zh-Hans.lproj/InfoPlist.strings

```diff
@@ -57,7 +57,7 @@
 "Safari extension" = "Safari 扩展";
 "ACE Archive" = "ACE 归档";
 "XIP Secure Archive" = "XIP 安全归档";
-"Apple Archive" = "Apple Archive";
```

Comment:
This one I'm not sure if should be not translated, since AAR (the extension) stands for Apple ARchive.

Reply:
it could be Android Archive too, and you think? ;)